build/lsof/patches/genunix.patch

```diff
@@ -0,0 +1,12 @@
+diff -wpruN '--exclude=*.orig' a~/dialects/sun/dnode.c a/dialects/sun/dnode.c
+--- a~/dialects/sun/dnode.c	1970-01-01 00:00:00
++++ a/dialects/sun/dnode.c	1970-01-01 00:00:00
+@@ -88,7 +88,7 @@ static	int	Sockfs_ctfs = 0;	/* CTF initi
+ 					 * sockfs */
+ 
+ #  if	defined(_LP64)
+-#define	SOCKFS_MOD_FORMAT "/kernel/fs/%s/sockfs"
++#define	SOCKFS_MOD_FORMAT "/kernel/%s/genunix"
+ #  else	/* !defined(_LP64) */
+ #define	SOCKFS_MOD_FORMAT "/kernel/fs/sockfs"
```

Comment:
This is the fallback for when lsof is !64-bit right?  Should it also grab it from genunix in that case?

Reply:
Yes, but a 32-bit lsof is for a 32-bit kernel, which we don't have. `/kernel/fs/sockfs` does not exist, and nor does `/kernel/genunix` so I did not bother changing one non-existent path to another.